### DISCUSSION TITLE
如何修改网站图标呢？

### DISCUSSION THREAD

**Discussion by xlztong** [zdz/ServerStatus-Rust#126] [General]
如题，感觉这个爬虫图标太丑了。。
![image](https://user-images.githubusercontent.com/59363725/208283947-0b5770e0-542e-4ecd-8f47-c0a21159cfe8.png)

**Comment by zdz** [comment]
https://github.com/zdz/ServerStatus-Rust/tree/master/web/ 替换一下，然后重新编译一下 `server`，或部署一下前端  https://github.com/zdz/ServerStatus-Rust/discussions/37

https://github.com/zdz/ServerStatus-Rust/blob/f5aaf7c6b5465e317e2412b6c6ce70a84229af42/web/index.html#L6
